src/map/atcommand.h

```diff
@@ -79,8 +79,10 @@ struct AtCommandInfo {
 struct atcmd_binding_data {
 	char command[ATCOMMAND_LENGTH];
 	char npc_event[ATCOMMAND_LENGTH];
-	int group_lv;
-	int group_lv_char;
+	int group_lv; // DEPRECATED
+	int group_lv_char; // DEPRECATED
+	char *at_groups; // quick @commands "can-use" lookup
```

Comment:
Hmm you're right, I didn't notice we already have that in use.

I still don't think it's a really good design (mostly because we already had bugs in the past, caused by this kind of array pointers being used as normal pointers), but it can stay for now.

Side note: you can access an already-known index of a VECTOR just like you can with an array (i.e. `VECTOR_INDEX(vector, i)` - and yes, it's uglier, but prevents people from accessing it as `*vector`, thinking it only has one record).

In any case, feel free to ignore this for now, it's beyond the scope of this pull request.